dpnp/dpnp_algo/dpnp_algo_mathematical.pyx

```diff
@@ -339,8 +339,38 @@ cpdef dparray dpnp_negative(dpnp_descriptor x1):
     return call_fptr_1in_1out(DPNP_FN_NEGATIVE, x1, x1.shape)
 
 
-cpdef dparray dpnp_power(object x1_obj, object x2_obj, object dtype=None, dparray out=None, object where=True):
-    return call_fptr_2in_1out(DPNP_FN_POWER, x1_obj, x2_obj, dtype=dtype, out=out, where=where)
+cpdef dparray dpnp_power(utils.dpnp_descriptor x1_obj, utils.dpnp_descriptor x2_obj, object dtype=None, dparray out=None, object where=True):
```

Comment:
Why we can't use the same as `call_fptr_1in_1out` (PR #775) with `call_fptr_2in_1out`?